docs/api/exceptions/dtexception.rst

```diff
@@ -0,0 +1,5 @@
+
+.. xexc:: datatable.exceptions.DtException
+    :src: --
+
+    Base class for all other exceptions raised by ``datatable``.
```

Comment:
The word "other" is kind of confusing here, because there is no other info on this page. What if we just remove it?